go/store/skip/list.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	maxHeight  = 9
-	maxCount   = math.MaxUint32 - 1
+	maxCount   = math.MaxInt - 1
```

Comment:
I think this might be better as:

```suggestion
	maxCount   = math.MaxInt32 - 1
```

for now. The structure will not work with a number of entries > `MaxUint32`, so we don't want to set it to `MaxInt - 1` on a 64-bit platform.

For now we can give up the two billion possible extra entries on 64-bit in order to get this working on 32-bit.